### ISSUE TITLE
Add step with no speech bubbles right before the architecture diagram step

### ISSUE THREAD

**Issue by crcastle** [crcastle/friends-from-the-trail-demo#3]
This step is when most demoers do the mobile app deploy. It would be nice to have a clean screen at this point for the QR code and URL to appear on.

Feedback from one demoer:

> Would have been helpful to have one more “blank” before the Architecture diagram in the series of speech bubbles - usually when I would present I would walk through all the speech bubbles then deploy the mobile app with the QR code and it would be helpful to have the web app waiting on a blank for when all the selfies came through
